### PR TITLE
Update changelog for v2

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,99 @@
 -   Upgrade `keycloak-js` to v26
 -   Remove prop-types
 
+**Breaking changes**
+
+The setup has been drastically simplified:
+
+```diff
+-import React, { useState, useRef, useEffect } from 'react';
+import React from 'react';
```
Comment (slax57) [inline CHANGELOG.md]:
```suggestion
+import React from 'react';
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,99 @@
...
+import { Admin } from 'react-admin';
+import simpleRestProvider from 'ra-data-simple-rest';
+import Keycloak, {
+    KeycloakConfig,
+    KeycloakTokenParsed,
+    KeycloakInitOptions,
+} from 'keycloak-js';
+import { keycloakAuthProvider, httpClient } from 'ra-keycloak';
```
Comment (slax57) [inline CHANGELOG.md]:
```suggestion
-import { keycloakAuthProvider, httpClient } from 'ra-keycloak';
+import { keycloakAuthProvider, httpClient, LoginPage } from 'ra-keycloak';
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,99 @@
...
+
+const config: KeycloakConfig = {
+    url: '$KEYCLOAK_URL',
+    realm: '$KEYCLOAK_REALM',
+    clientId: '$KEYCLOAK_CLIENT_ID',
+};
+
+const initOptions: KeycloakInitOptions = { onLoad: 'check-sso' };
```
Comment (slax57) [inline CHANGELOG.md]: The default was `{ onLoad: 'login-required' }` in the previous version. Encouraging moving to `{ onLoad: 'check-sso' }` is a good move IMO, but then you require the `LoginPage` too.

```suggestion
-const initOptions: KeycloakInitOptions = { onLoad: 'login-required' };
+const initOptions: KeycloakInitOptions = { onLoad: 'check-sso' };
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,99 @@
...
+-    }, [keycloak]);
+-
+-    // hide the admin until the keycloak client is ready
+-    if (!keycloak) return <p>Loading...</p>;
+
+     return (
+        <Admin
+            authProvider={authProvider.current}
+            dataProvider={dataProvider.current}
+        >
```
Comment (slax57) [inline CHANGELOG.md]:
```suggestion
        <Admin
            authProvider={authProvider.current}
            dataProvider={dataProvider.current}
+           loginPage={LoginPage}
        >
```

**Review by slax57 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (djhi) [reply CHANGELOG.md]: Not really needed IMO

Comment (djhi) [reply CHANGELOG.md]: Yes it was in the documentation but it wasn't required

**Review by slax57 — APPROVED** [review]

**Pull request merged**